Remove text shadow for collection action buttons

PunitGr opened the pull request:
Fixes #5575 

### Screenshot
![Screenshot 2019-06-03 at 9 53 22 PM](https://user-images.githubusercontent.com/9111111/58830454-a2e30680-864a-11e9-97a8-01067cad61f8.png)

Updated style for buttons to remove `text-shadow` for `Buttons` on CollectionEdit.

kumar303 commented:
Please make sure we address the accessibility concerns when removing the text-shadow. More info: https://github.com/mozilla/addons-frontend/issues/5575#issuecomment-498346164

PunitGr commented:
@willdurand Cool, let me try that.

PunitGr commented:
@willdurand I removed the `text-shadow` but as mentioned by @kumar303 the button was a bit blurry.
So, I added `letter-spacing` instead. Please refer to the attached screenshot above. But it is same as adding inconsistency in the buttons.

willdurand commented on src/ui/components/Button/styles.scss:
```diff
@@ -78,11 +78,12 @@ $default-font-size: 13px;
-      // See: https://github.com/FirefoxUX/photon/issues/264
-      text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
+      // Small text is difficult to read against the bright Photon colors
+      // since it is blurry
```
Could you add a period at the end of this sentence?

willdurand commented on src/ui/components/Button/styles.scss:
```diff
@@ -78,11 +78,12 @@ $default-font-size: 13px;
+      // since it is blurry
+      // This isn't (yet) part of the spec but added this because
+      // it improves accessibility.
+      // https://github.com/FirefoxUX/photon/issues/264#issuecomment-364927535
```
Could you add `See: ` before the URL?

PunitGr commented:
Hey, @willdurand. Thank you. 👍
